### PR TITLE
Fix formatting for payload attributes

### PR DESCRIPTION
**Pull request by ChrisCoe**
Before this change, certain attributes in the span/trace would not store anything because of bugs in formatting. 

@reyang

### CODE REVIEW

**Comment by ChrisCoe** [comment]
I fixed the string conversions for the attributes. @reyang

File: exporter/azure_monitor/azure_monitor.go [hunk 1]
```diff
@@ -63,14 +63,14 @@ func (exporter *AzureTraceExporter) ExportSpan(sd *trace.SpanData) {
 			Success : true,
 		}
 		if _, isIncluded := sd.Attributes["http.method"]; isIncluded {
-			currentData.Name = sd.Attributes["http.method"].(string)
+			currentData.Name = fmt.Sprintf("%s", sd.Attributes["http.method"])
 		}
 		if _, isIncluded := sd.Attributes["http.url"]; isIncluded {
-			currentData.Name = currentData.Name + " " + sd.Attributes["http.url"].(string)
-			currentData.Url = sd.Attributes["'http.url"].(string)
+			currentData.Name = currentData.Name + " " + fmt.Sprintf("%s", sd.Attributes["http.url"])
```
Comment (reyang) [inline exporter/azure_monitor/azure_monitor.go]: This probably should be `fmt.Sprintf("%s %s", currentData.Name, sd.Attributes["http.url"])`.

**Review by reyang — APPROVED** [review]
LGTM except for one format where there is a mixed use of string concatenation and formatting.

File: exporter/azure_monitor/azure_monitor.go (hunk 1 above)
Comment (ChrisCoe) [reply exporter/azure_monitor/azure_monitor.go]: I agree. This is cleaner. Will make change soon.

**Pull request merged**